tests/lib/rules/no-unused-prop-types.js

```diff
@@ -1387,15 +1319,16 @@ ruleTester.run('no-unused-prop-types', rule, {
       code: [
         'const PagingBlock = function(props) {',
         '  return (',
-        '    <SomeChild {...props} />',
+        '    <span>{...props}</span>',
```

Comment:
The fix was easy. Changing type from JSXChildSpread to JSXAttribute spread...Thanks for catching this. I will go read about JSXChildSpread though... maybe it is some beta feature...